web-common-marlowe/src/Marlowe/Extended/Template.purs

```diff
@@ -0,0 +1,9 @@
+module Marlowe.Extended.Template where
+
+import Marlowe.Extended (Contract)
+import Marlowe.Extended.Metadata (MetaData)
+
+type ContractTemplate
+  = { metaData :: MetaData
+    , extendedContract :: Contract
+    }
```

Comment:
I think it would make most sense to move `Template` and `Fillable` classes and instances here too since we are at it, right? And we could move this type synonym to `Metadata` module actually, it is called `ContractTemplate`, but it is actually `MetaData` plus `ExtendedMarlowe` (which has templates, so the name is not bad, but I think it probably fits better in Metadata module)

Reply:
I basically moved it to it's own file because it caused cyclic dependency. If I'd known that it was Metadata related I wouldn't had created the Template module :P. For the next PR I'll add a comments to the changes that we need to make, but I'll concentrate a little bit more in the dashboard functionality.